cpptests/common.h

```diff
@@ -73,14 +76,14 @@ class ConnectError : public ClientError {
 
 class IOError : public ClientError {
 public:
-    IOError() : ClientError(){}
-    IOError(const char *s) : ClientError(s) {}
+    IOError() : ClientError() {}
+    IOError(const char *what) : ClientError(what) {}
 };
 
 class TimeoutError : public ClientError {
 public:
     TimeoutError() : ClientError("timed out") {}
-    TimeoutError(const char *s) : ClientError(s) {}
+    TimeoutError(const char *what) : ClientError(what) {}
```

Comment:
Why the renaming? I mean, it might be a bit more intuitive...

Reply:
Different names for different exception types